worker/main.go

```diff
@@ -60,10 +62,15 @@ func GetHealth(w http.ResponseWriter, r *http.Request) {
 
 }
 
+func UnRegisterRequest(w http.ResponseWriter, r *http.Request) {
+	// TODO check if this worker is still processing a graph and when done unregister
+	unregister()
+}
+
 func register() {
 	for {
 		options := grequests.RequestOptions{
-			JSON:    map[string]string{"address": getOwnURL()},
+			JSON:    map[string]string{"address": getOwnURL(), "instanceId": conf.Own.Instanceid},
```

Comment:
Is het misschien handig om nu dan ook de workers bij te houden in de master op basis van instanceId ipv adres?

Reply:
we moeten adres en instanceId bij houden denk ik adres is nodig om graphs to process heen te sturen en instanceId hebben we nodig om een worker weer te kunnen stoppen